python/PyQt6/core/auto_generated/sensor/qgsiodevicesensor.sip.in

```diff
@@ -225,6 +225,18 @@ Returns the serial port the sensor connects to.
 Sets the serial port the sensor connects to.
 
 :param portName: the port name (e.g. COM4)
+%End
+
+    QString baudrate() const;
```

Comment:
@jtornero , when editing C++ files via github, auto-generated sip files are not updated (that'll create a CI failure).

Either manually edit the sip.in files (this way is prone to errors) or update your local branch to catch the edits you did, then run ./scripts/sipify_all.sh in the root folder of your QGIS source tree.